cmd/evaluate.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// // Counts how many of the pre-defined buzzwords are included in the article title
```

Comment:
Yes it is, but I guess it can be removed, due to it being completely basic and useless currently. We might choose to do this with AI right off the start